admin-dev/themes/new-theme/js/pages/order/view/order-view-page.js

```diff
@@ -204,6 +204,29 @@ export default class OrderViewPage {
     });
   }
 
+  listenForRefund() {
+    $(OrderViewPageMap.displayPartialRefundBtn).on('click', () => {
+      this.orderProductRenderer.moveProductsPanelToRefundPosition();
+      $(OrderViewPageMap.toggleStandardRefundForm).hide();
```

Comment:
Can you keep the same order for `toggleStandardRefundForm`, `togglePartialRefundForm` and `actionColumnElements` in the three functions?

Reply:
Actually I can't, I added a comment to explain why `// Always start by hiding elements then show the others, since some elements are common`